data/assets/examples/screenspacerenderable/screenspacetimevaryingimageonline/data/dummydata.json

```diff
@@ -0,0 +1,16 @@
+{
+ "files": [
+  {
+   "timestamp": "2024-05-10 15:00:00.0",
+   "url": "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
+  },
+  {
+   "timestamp": "2024-05-10 15:10:00.0",
+   "url": "https://images.squarespace-cdn.com/content/v1/5a769958b0786976fb16bcf6/1542282853508-9NJRS30YDM5TSXIE41CE/openspace-color-on-black.png?format=2500w"
```

Comment:
Would be better to use another image from our data server to ensure that the link will stay alive, when do do changes on SquareSpace, for example.

I added a second image to the examples folder: https://data.openspaceproject.com/examples/renderableplaneimageonline_2.jpg